db/migrate/20170616152705_set_codelocation_status_to_nil.rb

```diff
@@ -0,0 +1,93 @@
+class SetCodelocationStatusToNil < ActiveRecord::Migration
```

Comment:
@PDegenPortnoy had asked to have this called in a migration to allow for the ability to rollback the changes if necessary.  @PDegenPortnoy is this what you expected?